docs/algorand/for-end-users/Algorand-Standard-Assets-support.mdx

```diff
@@ -0,0 +1,147 @@
+
+<li>Input your ASA ID or its symbol (EURE or 12400859) 
+
+<img src={addingEuretowallet} alt="adding EURe to Wallet" />
+
+If you are having trouble finding this particular ASA - check if you have changed the network in the Wallet to Algorand testnet (_in the upper right corner_).
+</li> 
+<li>Wait until EURe with value of 0 appears on your list of assets in your account</li> 
+</ol>
+Now, you can receive EURe on your account using the testnet dispenser! 
+
+<h3>How to get testnet EURe </h3>
+
+For the purpose of this tutorial we will focus only on testnet Euros which is default ASA available on official Algorand Testnet dispenser.
+
+Go through the steps below to acquire your first tesnet EURe ASA.
+
+<ol> 
+<li>Go to <a href="https://testnet.algoexplorer.io/dispenser">https://testnet.algoexplorer.io/dispenser</a></li> 
+<li>Select Testnet Euros from the drop-down list as shown in the image below</li> 
+<img src={asadispenser} className="img-full" alt="getting ASA from dispenser" />
+<li>Paste your Algorand wallet address</li> 
+<li>Complete the required anti-bot verification</li> 
+<li>Click Dispense button to receive your Euros</li> 
+</ol> 
+
+As you received your testnet Euros now you are ready to start interacting with the Milkomeda Alrgorand Rollup bridge. How cool is that! 
+
+<h3>Wrapping Algorand Standard Assets with Milkomeda Bridge</h3>
+
+
+Go to the Milkomeda A1 bridge dApp at https://algorand-bridge-dev.milkomeda.com/ and select the token that you would like to transfer. To start wrapping your assets, choose **Algorand to Milkomeda** direction from Network Origin dropdown button. Then **select the appropriate token (EURe)** from Select Token dropdown button.
+
+<img src={EURewrappingchoice} className="img-full" alt="Wrapping EURe" />
+
+:::caution
+Remember to have a minimum amount of ALGO tokens in your Algorand wallet to move EURe as it will be used for the transaction fee on mainchain. 
+:::
+
+**Enter how many test EURe** you want to transfer to the A1 rollup. 
+
+**Click “Next”** to view a transaction summary, complete with “Estimated Fees” and an “Estimated Wrapping Confirmation Time”.
+
+<img src={EURewrappingconfirmation} className="img-full" alt="Confirm EUR Wrapping" />
+
+**Click “Sign and Send”,** enter your MyAlgo wallet password, confirm transaction and then watch as your tokens get sent to the bridge, wrapped, and sent to your Metamask wallet on the Milkomeda Algorand Rollup! The process should take around 20 seconds to confirm.
+
+:::info
+The transfer only requires a total of 2 confirmations, which should take approximately 20 seconds (4.5 seconds for Algorand and around 14 seconds for the batch of the Rollup). 
+:::
+You can now watch your transaction pass through the bridge.
+<img src={ASAwrappedconfirmations} className="img-full" alt="ASA wrapping blockchain confirmations" />
+
+Once the transaction is confirmed, we recommend adding the ASA to the Metamask wallet.
```

Comment:
```suggestion
Once the transaction is confirmed you'll need to add the ASA to your MetaMask wallet.
```

Reply:
> 

@welshiewelshie , you don't need to - but you should to see it in metamask - it's not like with ASA in Algo wallets.